controller/controller.js

```diff
@@ -0,0 +1,64 @@
+import logger from "../config/logger";
+import Note from "../models/notes";
```

Comment:
We can set config and models as common folders for absolute paths. We will use them pretty often, so it will be better to use them without additional dots.
```
import logger from "config/logger";
import Note from "models/notes";
```

Reply:
But also we can move `models` to the `db` folder and make db as absolute.